app/components/blacklight/icons/icon_component.rb

```diff
@@ -6,15 +6,27 @@ module Icons
     # You can override the default svg by setting:
     #   Blacklight::Icons::ListComponent.svg = '<svg>your SVG here</svg>'
     class IconComponent < ::ViewComponent::Base
-      def initialize(svg: nil)
+      # rubocop:disable Metrics/ParameterLists
+      def initialize(svg: nil, component: :span, name: nil, label: nil, aria_hidden: nil, classes: '', **options)
```

Comment:
I think `component` here is confusing as it's not a view component.  How about calling it `tag:` or `wrapper:` instead?